example_sample_sheets/miseq_single_end_run/SampleSheet.csv

```diff
@@ -0,0 +1,23 @@
+[Header]
```

Comment:
We did not have a miseq single end run before, so adding this made it easier to test those features in the gui.